### DISCUSSION TITLE
Some firearms should automatically drop magazines

### DISCUSSION THREAD

**Discussion by PaulRitter** [space-wizards/space-station-14#3994] [Feature Request]
And right clicking to eject magazine takes an enormous amount of time. It goes to vanilla guns mostly because the eris guns auto-drop magazines

Reported by @"Herbert Falsevor#3037"

**Comment by PJB3005** [comment]
This is a feature. Auto-ejecting guns should be high-end stuff like nuke ops weapons.

Making manual ejections less painful (no right clicking) would still be good though.
